quinn-proto/src/connection/spaces.rs

```diff
@@ -695,6 +699,17 @@ impl PendingAcks {
     pub(super) fn ranges(&self) -> &ArrayRangeSet {
         &self.ranges
     }
+
+    /// Should be called immediately before a non-probing packet is composed
+    pub(super) fn on_send_scheduled(&mut self) {
+        // If we're going to send a packet anyway, and we've received a significant number of
+        // non-ACK-eliciting packets, then include an ACK to help the peer perform timely loss
+        // detection even if they're not sending any ACK-eliciting packets themselves.
+        const LAZY_ACK_THRESHOLD: u64 = 10;
```

Comment:
This seems fairly low to me?

Reply:
My thinking is that the impact is modest (ACK frames are usually small), so returns probably diminish rapidly. We're also constraining this to whenever the local endpoint has something to send regardless, so the actual rate may be lower.

I didn't think super hard about it, though. Want to propose another figure?

Reply:
I guess it's fine.